tests/test_local_pipeline.py

```diff
@@ -23,3 +25,13 @@ def test_doc(self):
         self.assertEqual(ta.get_score, 1.0)
 
         self.assertEqual(ta.get_text, "Hello,  how are you.\n\n\n I am doing fine")
+
+    def test_doc_illigal_characters(self):
+        ta = self.lp.doc("Hillary Clinton�s Candidacy Reveals Generational Schism Among Women https://t.co/6u3lmN7nIL")
```

Comment:
Semaphore output says the encoding needs to be specified. Try `u"Hillary Clinton�s Candidacy Reveals Generational Schism Among Women https://t.co/6u3lmN7nIL"`, u is prefix for unicode string.
(I think it is another Python 2 issue, I tried encoding the string on Python 3 and Python 2 and only Python 2 gave error when "u" wasn't added)

Reply:
Ah good catch! Thanks!